lib/utils/exit_program.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+Future<bool?> showExitWarning(BuildContext context) async {
+  if (Navigator.canPop(context)) {
+    Navigator.pop(context);
+  } else {
+    showDialog<bool>(
```

Comment:
Not returning the user choice
use return await here to return the value to the previous page and do the necessary

Currently pressing yes is not exiting the app

Reply:
Fixed [here](https://github.com/gdsc-nits-org/efficacy_admin/pull/25/commits/e305f928eca46451eca64c901c3a38941f44b254)